doc_source/understanding-ssh-in-amazon-lightsail.md

```diff
@@ -21,8 +21,10 @@ You can also manage SSH keys from the **Account** page\.
 
 In Lightsail, the SSH key pairs you use are AWS Region specific\. Lightsail creates a **Default** key in each region\. If you want to create a new key pair to use with your instances, you'll need to create or upload a new key pair in each region where you have Lightsail instances\. After you create a new key pair, you can upload the same key to multiple regions\. For more information, see [Regions and Availability Zones in Lightsail](understanding-regions-and-availability-zones-in-amazon-lightsail.md)\.
 
+NOTE: If you are using a Mac with 10.15.5 (Catalina) - You can connect to your AWS Lightsail Instance by simply downloading the .pem file from the Account page. After that, in order to ensure that the SSH connection is working, you will need to first download it to a directory and from there changemode or chmod 600 the file. From there, while working in the same directory, the following command will ensure that you can connect without a problem: ssh -i "....pem" username@publicIPAddress
+
```

Comment:
Update: This tutorial also works on Ubuntu (16.04) just fine. Following the same tutorial as the one for Mac works!

Reply:
I am using `.pem` files to access Lightsail instances from Windows 10, using built-in OpenSSH clients, as well as the one in WSL (I think I used both Debian 9 and Ubuntu 20 in WSL, both working fine with `.pem`). So it appears to be a general technique. Maybe no need to mention OS X version number or maybe even OS X at all?

Reply:
I remembered when I first opened up this pull request and I was following the AWS documentation for Mac but it was not clear enough. I followed it to a T but it was not until I used the ssh method to connect to an EC2 instance that it finally worked. I also tried on Ubuntu but it was just not working so I figured that I mention that in case anyone else were experiencing the same issue